.goreleaser.yaml

```diff
@@ -111,6 +111,9 @@ docker_manifests:
   image_templates:
   - tensorchord/envd-from-scratch:v{{ .Version }}-amd64
   - tensorchord/envd-from-scratch:v{{ .Version }}-arm64v8
+release:
+  skip_upload: true
```

Comment:
I am wondering why we do not upload it with goreleaser.

Reply:
1. This is because the format of the changelog uploaded using goreleaser is different from that of https://github.com/tensorchord/envd/blob/main/CHANGELOG.md.
2. @terrytangyuan said we need reuse this script: https://github.com/tensorchord/envd/blob/main/hack/changelog.sh https://github.com/tensorchord/envd/issues/1140#issuecomment-1300231365
3. We think we need put a redirect link in the release page
https://github.com/tensorchord/envd/issues/1140#issuecomment-1300754617

Reply:
Gotcha, thanks for your explanation!